pmd-core/src/main/java/net/sourceforge/pmd/cpd/CPDConfiguration.java

```diff
@@ -131,7 +131,7 @@
     @Parameter(names = { "--help", "-h" }, description = "Print help text", required = false, help = true)
     private boolean help;
 
-    @Parameter(names = { "--failOnViolation", "-failOnViolation" }, arity = 1,
+    @Parameter(names = { "--fail-on-violation", "--failOnViolation" }, arity = 1,
```

Comment:
We should keep the old variant alive...

```suggestion
    @Parameter(names = { "--fail-on-violation", "--failOnViolation", "-failOnViolation" }, arity = 1,
```